snap/snapcraft.yaml

```diff
@@ -16,29 +16,32 @@ plugs:
 
 environment:
   JAVA_HOME: $SNAP/usr/lib/jvm/java-11-openjdk-amd64
-  PATH: $JAVA_HOME/bin:$PATH
+  PATH: $JAVA_HOME/bin:$PATH:$SNAP
+  KUBECONFIG: $SNAP_REAL_HOME/.kube/config
+  SPARK_HOME: $SNAP
+  SPARK_CONFS: $SNAP/lib/python3.10/site-packages/spark8t/resources/conf/
```

Comment:
OK, indeed this should be re-visited. However especially since it requires changes on 2 repos, I agree to leave it for a little later, and now just let's have a "green chain" that's primarily correct, even if there's further cosmetics 2 B done.

Reply:
I've added a reference of this thread to https://warthogs.atlassian.net/browse/DPE-1884